src/libsync/owncloudpropagator.cpp

```diff
@@ -388,6 +395,9 @@ void OwncloudPropagator::start(const SyncFileItemVector& items)
                 currentDirJob->append(dir);
             }
             directories.push(qMakePair(item->destination() + "/" , dir));
+        } else if (enableScheduledRequests
+              && (item->_instruction == CSYNC_INSTRUCTION_NEW || item->_instruction == CSYNC_INSTRUCTION_SYNC)) {
+            filesJob->append(item);
```

Comment:
@mrow4a Okay, I didn't see that. (btw, _runningNow is gone, so this branch doesn't compile anymore) In that case the second problem can indeed not happen.